handler/core/src/main/java/io/knotx/fragments/handler/FragmentsHandler.java

```diff
@@ -51,14 +51,14 @@
 
   private final FragmentsEngine engine;
   private final RequestContextEngine requestContextEngine;
-  private final TaskBuilder taskBuilder;
+  private final TaskManager taskProvider;
```

Comment:
Some inconsequence. If the class is `TaskManager`, the variable should be `taskManager`. But I think that `TaskProvider` is a better name here.

Reply:
Sure but we use all *Provider to deliver more implementations of particular interface, such as:
- `TaskProvider`
- `NodeProvider`
Then we can specify:
- `ConfigurationTaskProviderFactory`, not `ConfigurationTaskFactoryFactory ;)

TaskManager loads all task providers.